rest_framework_json_api/pagination.py

```diff
@@ -109,6 +109,9 @@ class PageNumberPagination(BasePagination):
     A json-api compatible pagination format
     """
 
+    page_size_query_param = 'page_size'
+    max_page_size = 100
+
```

Comment:
I moved those from the settings. Pagination configuration from Drf >= 3.1 should be set in the pagination class. Putting them in the view or settings as per Drf < 3.1 now produces a 'deprecation warning' and I think it will produce an error from Drf 3.3.